### PR TITLE
Lesson 09 Mailroom Assignment

### PR DESCRIPTION
**Pull request by jeanruggiero**

### CODE REVIEW

File: students/jeanruggiero/Lesson09/mailroom5.py [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+    @property
+    def average_donation(self):
+        return statistics.mean(self._donations)
+
+    def add_donation(self, amount):
+        try:
+            amount = float(amount)
+        except ValueError:
+            print('Please enter a number value for donation amount.')
+        self._donations.append(amount)
```
Comment (natasha-aleksandrova) [inline students/jeanruggiero/Lesson09/mailroom5.py]: I think you want this inside the `else` block?

**Comment by natasha-aleksandrova** [comment]
@jeanruggiero can you sync your fork please? I can't merge this until you do, thanks!

**Comment by natasha-aleksandrova** [comment]
please re-open after syncing.

**Comment by jeanruggiero** [comment]
Hi Natasha,

I think I've merged my branch back in. I submitted pull request #1382 which
contains this assignment and the lesson 10 assignment. Did I do the
branching wrong?

https://github.com/UWPCE-PythonCert-ClassRepos/Self_Paced-Online/pull/1382

Jean

**Pull request closed without merging**